Fix tests in ReleaseStack_WCProductTest

The `testUpdateProduct()` is failing in `ReleaseStack_WCProductTest` with the following error:
```
java.lang.AssertionError: OnProductUpdated has unexpected error: INVALID_PARAM, Invalid parameter(s): tax_status
at org.wordpress.android.fluxc.release.ReleaseStack_WCProductTest.onProductUpdated(ReleaseStack_WCProductTest.kt:445) 
```

--- a/example/src/androidTest/java/org/wordpress/android/fluxc/release/ReleaseStack_WCProductTest.kt
+++ b/example/src/androidTest/java/org/wordpress/android/fluxc/release/ReleaseStack_WCProductTest.kt
@@ -56,6 +56,9 @@ class ReleaseStack_WCProductTest : ReleaseStack_WCBase() {
     private val productModel = WCProductModel(8).apply {
         remoteProductId = BuildConfig.TEST_WC_PRODUCT_ID.toLong()
         dateCreated = "2018-04-20T15:45:14Z"
+        taxStatus = "taxable"
+        stockStatus = "instock"
+        backorders = "yes"
     }
     private val productModelWithVariations = WCProductModel(8).apply {
         remoteProductId = BuildConfig.TEST_WC_PRODUCT_WITH_VARIATIONS_ID.toLong()

